src/features/import/hooks/useDateConfig.ts

```diff
@@ -62,7 +62,17 @@ export default function useDateConfig(column: DateColumn, columnIndex: number) {
     personNumberFormats.push('se', 'dk', 'no');
   }
 
-  const dateFormats = ['YYYY-MM-DD', 'YY-MM-DD', 'MM-DD-YYYY'];
+  enum DateFormats {
+    'YYYY-MM-DD' = 'YYYY-MM-DD',
+    'YY-MM-DD' = 'YY-MM-DD',
+    'MM-DD-YYYY' = 'MM-DD-YYYY',
+  }
```

Comment:
Why make this an enum instead of just keeping the array?

Reply:
same as above, probably just a spinny head :)